server/src/main/java/org/elasticsearch/cluster/RestoreInProgress.java

```diff
@@ -342,10 +359,21 @@ public RestoreInProgress(StreamInput in) throws IOException {
             uuid = in.readString();
             Snapshot snapshot = new Snapshot(in);
             State state = State.fromValue(in.readByte());
+            boolean silent = true;
```

Comment:
Suggest a comment about why this is the default in BwC situations (i.e. older versions never did this logging anyway)

also a style nit: I have a slight preference for doing this assignment in an else branch, especially if we're adding a comment about it.